docs/Skill_Tutorial.md

```diff
@@ -73,14 +73,14 @@ In your editor window write:
 roses are red
 SUSI is a hack
 ```
-This defines one simple intent: to answer on "roses are red" the phrase "SUSI is a hack". The other lines mean:
+This defines one light intent: to answer on "roses are red" the phrase "SUSI is a hack". The other lines mean:
 * all lines starting with `#` are comment lines and are ignored.
 * all other text lines define Skills. Skills are separated by empty lines. Comment and section declaration modifiers also count as empty lines and separate Skills.
 
 Now you can test the new intent:
 * send the following query to SUSI: "roses are red"
 * SUSI.AI will answer with "SUSI is a hack".
-The Skill file is just a text file where two lines which are not separated by an empty line represent a conversation pattern.
+The Skill file is a text file where two lines which are not separated by an empty line represent a conversation pattern.
```

Comment:
Revert this change

Reply:
ON reverting tests won't pass as it is insensitive in language

Reply:
In that case, I am not sure about the flexibility of the Alex tool and if we should go with it.

Reply:
This tool is just to make language more sensitive and checks for any insensitive grammar 
For this line it would serve the same purpose as without the word 'just'